cineast-api/src/main/java/org/vitrivr/cineast/api/rest/handlers/actions/bool/FindDistinctElementsByColumnPostHandler.java

```diff
@@ -62,20 +54,34 @@ public DistinctElementsResult performPost(ColumnSpecification specification, Con
     LOGGER.trace("Retrieved unique values for {} in {} ms", specification.getTable() + "." + specification.getColumn(), watch.getTime(TimeUnit.MILLISECONDS));
     return new DistinctElementsResult("", distinct);
   }
-
+  
   @Override
   public Class<ColumnSpecification> inClass() {
     return ColumnSpecification.class;
   }
-
-
+  
+  
   @Override
   public Class<DistinctElementsResult> outClass() {
     return DistinctElementsResult.class;
   }
-
+  
   @Override
   public String route() {
     return ROUTE;
   }
+  
+  
+  @Override
+  public OpenApiDocumentation docs() {
+    return OpenApiBuilder.document()
+        .operation(op -> {
+          op.summary("Find all distinct elements of a given column");
+          op.description("TODO");
```

Comment:
I think for most handlers the description matches the summary so we can do the same thing here. One thing to note is that this handler caches results

Reply:
Thanks for the note, it's written in the new description.
I do think its important to have it there, as the description is where longer, well description, goes.
This is rather a hint for future handlers to include these and one might start an offensive to fully document the api...